### DISCUSSION TITLE
Global PYTHONPATH Variable

### DISCUSSION THREAD

**Discussion by aperloff** [CoffeaTeam/coffea-casa#137] [UNL tech support]
Is there a way to set the `PYTHONPATH` variable at startup. Otherwise we need to specify machine specific paths everyone in our code to use module paths like `utils.python.xxx`. Alternatively we would need to open a terminal each time we login, which could get annoying.

**Comment by oshadura** [comment]
I think it is very good remark... Could you share with me please your repository so I could have a quick look?
